zathura/landlock.c

```diff
@@ -0,0 +1,129 @@
+#define _GNU_SOURCE
```

Comment:
This file is missing the common comments.

Reply:
Not sure what you mean by common comments. It is included in the landlock macro now though, so it wont be included in non Linux builds

Reply:
```
/* SPDX-License-Identifier: Zlib */
```